src/plugins/data/server/ui_settings.ts

```diff
@@ -579,5 +579,21 @@ export function getUiSettings(
       schema: schema.number(),
       requiresPageReload: true,
     },
+    [UI_SETTINGS.SEARCH_EXCLUDED_DATA_TIERS]: {
+      name: i18n.translate('data.advancedSettings.searchExcludedDataTiers', {
+        defaultMessage: 'Excluded Data Tiers From Search',
```

Comment:
This should use title case:

```suggestion
        defaultMessage: 'Excluded data tiers from search',
```